development/db/structuredConditionals.rst

```diff
@@ -0,0 +1,504 @@
+===========================
+The boolean structure system
+===========================
```

Comment:
As long as the text of the heading. This obviously also applies to the line above the text.